File: src/main/java/com/mrpowergamerbr/loritta/utils/correios/EncomendaResponse.java
package com.mrpowergamerbr.loritta.utils.correios;

import lombok.Getter;

import java.util.List;

@Getter
public class EncomendaResponse {
	public String date;
	public String time;
	public String state;
	public List<PackageUpdate> locations;
	
	@Getter
	public static class PackageUpdate {
		public String state;
		public String reason;
		public String location;
		public String receiver;
		public String date;
		public String time;
	}
}

File: platforms/discord/legacy/src/main/java/com/mrpowergamerbr/loritta/utils/correios/EncomendaResponse.kt
package com.mrpowergamerbr.loritta.utils.correios

class EncomendaResponse {
	var date = "???"
	var time = "???"
	var state = "???"
	var error = "???"
	var locations: List<PackageUpdate> = listOf()

	class PackageUpdate {
		var state = "???"
		var reason = "???"
		var location = "???"
		var receiver = "???"
		var date = "???"
		var time = "???"
	}
}